src/site/_includes/partials/analytics.njk

```diff
@@ -10,12 +10,6 @@ ga('set', 'transport', 'beacon');
 ga('set', 'anonymizeIp', true);
 ga('set', 'page', window.location.pathname);
 ga('set', '{{ site.analytics.dimensions.TRACKING_VERSION }}', '{{ site.analytics.version}}');
-try {
-  ga('set', '{{ site.analytics.dimensions.NAVIGATION_TYPE }}', performance.getEntriesByType('navigation')[0].type.replace(/_/g, '-'));
-} catch (error) {
-  ga('set', '{{ site.analytics.dimensions.NAVIGATION_TYPE }}', '(not set)');
-}
 ga('set', '{{ site.analytics.dimensions.COLOR_SCHEME_PREFERENCE }}', window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light');
```

Comment:
I think it probably makes sense to move this line into the `analytics.js` file as well.

Reply:
Yeah I went back and forth on this one myself. Eventually, as it was simple I left it here just in case any other event (that may run before analytics.js ran) needed it. will move.